src/instructlab/model/evaluate.py

```diff
@@ -204,8 +208,7 @@ def qa_pairs_to_qna_to_avg_scores(qa_pairs: list[dict]) -> dict:
 )
 @click.option(
     "--benchmark",
-    type=click.Choice(list(BENCHMARK_TO_CLASS_MAP.keys())),
-    # case_sensitive=False,
+    type=click.Choice(["mmlu", "mmly_branch", "mt_branch", "mt_bench_branch"]),
```

Comment:
```suggestion
    type=click.Choice(["mmlu", "mmlu_branch", "mt_branch", "mt_bench_branch"]),
```

Reply:
Sigh ... And that's why ad-hoc string constants are bad and enums are better.